tests/rptest/tests/schema_registry_test.py

```diff
@@ -276,7 +276,7 @@ def test_schemas_types(self):
         result_raw = self._get_schemas_types()
         assert result_raw.status_code == requests.codes.ok
         result = result_raw.json()
-        assert result == ["AVRO"]
+        assert result == ["PROTOBUF", "AVRO"]
```

Comment:
nit: set(result) == set(["PROTOBUF", "AVRO"]) but probably doesn't matter much since it's sourced from a statically defined vector